Source/Shared/BPUtils.m

```diff
@@ -23,34 +23,29 @@ @implementation BPUtils
     const char *color;
 } Message;
 
+// These must match BPKind in BPUtils.h
 Message Messages[] = {
-    {" PASSED ", ANSI_COLOR_GREEN },
-    {" FAILED ", ANSI_COLOR_RED   },
-    {" TIMEOUT", ANSI_COLOR_YELLOW},
+    {"        ", ANSI_COLOR_RED   }, // 0
```

Comment:
It's so that the indices of the colors match the numbers in the struct... I guess we could change the enums to start at zero but it really should be the same structure. Let me think of a way to unify them.